sandbox/apparmor/apparmor.go

```diff
@@ -78,6 +78,7 @@ const (
 func setupConfCacheDirs(newrootdir string) {
 	ConfDir = filepath.Join(newrootdir, "/etc/apparmor.d")
 	CacheDir = filepath.Join(newrootdir, "/var/cache/apparmor")
+	InternalAbi30File = filepath.Join(filepath.Dir(dirs.SnapAppArmorDir), "3.0") // /var/lib/snapd/apparmor/3.0
```

Comment:
Nitpick, not sure the comment is required?

Reply:
I wanted to let anyone looking know what the path is like in practice without having to evaluate the expression in their mind.